roles/openshift_logging/templates/es.j2

```diff
@@ -95,6 +95,13 @@ spec:
               readOnly: true
             - name: elasticsearch-storage
               mountPath: /elasticsearch/persistent
+          readinessProbe:
+            exec:
+              command:
+              - "/usr/share/elasticsearch/probe/readiness.sh"
+            initialDelaySeconds: 5
+            timeoutSeconds: 5
```

Comment:
@rhcarvalho  my only worry is then this (taken from `man curl`):

````
 -m, --max-time <seconds>
              Maximum time in seconds that you allow the whole operation to take.  This is useful for preventing your batch jobs from hanging
              for  hours  due to slow networks or links going down.  Since 7.32.0, this option accepts decimal values, but the actual timeout
              will decrease in accuracy as the specified timeout increases in decimal precision.  See also the --connect-timeout option.

              If this option is used several times, the last one will be used.
````

Notice:  "`Since 7.32.0, this option accepts decimal values, ...`"

I only hope there will be always curl that can interpret decimal values for this option correctly.